subjects/java/checkpoints/config-protector/README.md

```diff
@@ -0,0 +1,67 @@
+## Config Protector
+
+### Instructions
+
+Create a class `ConfigProtector` that provides a method to hide sensitive data in a configuration file using regex. The method should replace sensitive values with asterisks. The configuration file will be provided as a string, and the keys for the sensitive data will be given in a list.
```

Comment:
Maybe we could highlight the word Regex and the types of the files as well, the string and the list.

Reply:
what do you mean by saying the types of the files?

Reply:
"... provided as a `string`, and the keys for the sensitive data will be given in a `list`."

Reply:
done